schema.graphql

```diff
@@ -78,7 +95,11 @@ type Session {
 	userId: Int!
 }
 
+"""
+The root subscription type which contains root level fields.
+"""
```

Comment:
Where is this comment from?

Reply:
`v1/gql/mod.rs` file

Reply:
Can it be removed since its not accurate